doc/admin/installation.md

```diff
@@ -3,66 +3,58 @@
 
-## Software Dependencies
-
-DAOS requires a C99-capable compiler, a Go compiler, and the scons build tool.
-Moreover, the DAOS stack leverages the following open source projects:
-
--   [*gRPC*](https://grpc.io/) provides a secured out-of-band channel for
-    DAOS administration.
-
--   [*PMDK*](https://github.com/pmem/pmdk.git) for persistent memory
-    programming.
-
--   [*SPDK*](http://spdk.io/) for userspace NVMe device access and management.
-
--   [*ISA-L*](https://github.com/01org/isa-l) and
-    [*ISA-L_Crypto*](https://github.com/01org/isa-l_crypto) for checksum and
-    erasure code
-
--   [*Argobots*](https://github.com/pmodels/argobots) for thread management.
+## Distribution Packages
 
--   [*hwloc*](https://github.com/open-mpi/hwloc) for discovering system devices,
-    detecting their NUMA node affinity and for CPU binding
+DAOS packages are available for CentOS7, openSUSE Leap 15 and Ubuntu 20.04.
+For other distribution, manual building is required (see next section).
 
--   [*libfabric*](https://github.com/ofiwg/libfabric) for detecting fabric
-    interfaces, providers, and connection management.
+### CentOS 7.9
 
-The DAOS build system can be configured to download and build any missing
-dependencies automatically.
+Configure the DAOS package repository as follows:
+``` bash
+$ sudo wget -O /etc/yum.repos.d/daos-packages.repo http://packages.daos.io/v1.2/CentOS7/packages/x86_64/daos_packages.repo
+```
 
-## Distribution Packages
+epel-release is required for both client and server:
+``` bash
+$ sudo yum install -y epel-release
+```
 
-DAOS RPM packaging is currently available, and DEB packaging is under
-development and will be available in a future DAOS release.
-Integration with the [Spack](https://spack.io/) package manager is also
-under consideration.
+To install the DAOS client packages:
+``` bash
+$ sudo yum install -y daos-client
+```
 
-### Installing DAOS from RPMs
+To install the DAOS server packages:
+``` bash
+$ sudo yum install -y daos-server
+```
 
-DAOS RPMs are available from the Intel&copy; Registration Center.
-Clicking the [Intel&copy; Registration Center](https://registrationcenter.intel.com/forms/?productid=3412)
-link will take you to the registration center, where you will create an account.
-After creating an account, the following files can be downloaded:
+Debug and source RPMs available in the respective [debug](http://packages.daos.io/v1.2/CentOS7/debug/x86_64/daos_debug.repo)
+and [source](http://packages.daos.io/v1.2/CentOS7/source/daos_source.repo)
+repositories.
 
-- daos_debug.tar - _debuginfo_ packages
-- daos_packages.tar - client and server main packages
-- daos_source.tar - source RPMs
+### openSUSE Leap 15
 
-Recommended steps after download:
+Add the DAOS package repository via zypper:
+``` bash
+$ sudo zypper ar http://packages.daos.io/v1.2/Leap15.2/packages x86_64/daos_packages
+$ sudo rpm --import http://packages.daos.io/RPM-GPG-KEY
+$ sudo zypper --non-interactive ref
+```
```

Comment:
This is still using the packages.daos.io rep, not the unpacked downloaded tarballs like the CentOS examples.